cypress/integration/view-and-manage-evidence.spec.ts

```diff
@@ -124,6 +124,11 @@ describe('Can view and manage evidence', () => {
       rejectionReason: 'some rejection reason',
     });
   });
+
+  it('can copy the page URL using button', () => {
```

Comment:
This could be renamed to `can view approved documents` because that's the functionality we're testing, and as part of that we check that the copy button is visible